x-pack/legacy/plugins/siem/cypress/README.md

```diff
@@ -1,61 +1,237 @@
 # Cypress Tests
```

Comment:
Note to PR reviewers: you may view a markdown-rendered version of the changes in this PR by clicking on the `...` in the `GitHub` diff and selecting [View file](  https://github.com/elastic/kibana/blob/cd0ce0f0c6f2e77cb04ef0fbe2e29fd93314dcb4/x-pack/legacy/plugins/siem/cypress/README.md)